### PR TITLE
Search Page

### LINKED ISSUE #40: Search Result Component
**Is your feature request related to a problem? Please describe.**
There is no method to render a search result currently. 

**Describe the solution you'd like**
Need to create a component to display a single search result

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. Setup search feature
2. Render results on the search page

## Purpose
To enable users to search for articles

## Approach
1. Created state to house the search results and passed it as props to necessary components
2. Created and adjusted interfaces to pass props as needed
2. Wrote a function to map over search results and render them
3. Used styled-components to style the page and articles

Closes #40

### CODE REVIEW

File: the-toad-tribune/src/SubApp.tsx [hunk 1]
```diff
@@ -132,9 +135,9 @@ const SubApp = () => {
   };
 
   useEffect(() => {
-    newsDataGrabber();
-    adsDataGrabber();
-    weatherDataGrabber();
+    // newsDataGrabber();
+    // adsDataGrabber();
+    // weatherDataGrabber();
```
Comment (Lap343) [inline the-toad-tribune/src/SubApp.tsx:140]: is this still needed?

**Review by Lap343 — APPROVED** [review]
Very nice job

**Review by Gjhnsn — APPROVED** [review]
Nice work!

File: the-toad-tribune/src/SubApp.tsx (hunk 1 above)
Comment (krferrell) [reply the-toad-tribune/src/SubApp.tsx:140]: Yes. Just commenting them out so we don't blow through our API keys. I will add a comment in the code letting the reviewers know for next time

**Pull request merged**